data/net/minecraft/world/level/levelgen/feature/DeltaFeature.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/world/level/levelgen/feature/DeltaFeature
+	METHOD isClear (Lnet/minecraft/world/level/LevelAccessor;Lnet/minecraft/core/BlockPos;Lnet/minecraft/world/level/levelgen/feature/configurations/DeltaFeatureConfiguration;)Z
+		ARG 0 level
+		ARG 1 pos
+		ARG 2 deltaFeatureConfiguration
```

Comment:
Same as above. (For reference, most mcp names for these will be `config`. Possibly worth standardizing to `configuration`  across the board?